src/styles/reset.scss

```
@@ -0,0 +1,6 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
```

move it to the `html` selector
```suggestion
```
```suggestion
  font-family: Arial, Helvetica, sans-serif;
  font-size: 30px;
```